samples/11.qnamaker/src/main/java/com/microsoft/bot/sample/qnamaker/QnABot.java

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.sample.qnamaker;
+
+import com.microsoft.bot.ai.qna.QnAMaker;
+import com.microsoft.bot.ai.qna.QnAMakerEndpoint;
+import com.microsoft.bot.ai.qna.QnAMakerOptions;
+import com.microsoft.bot.builder.ActivityHandler;
+import com.microsoft.bot.builder.MessageFactory;
+import com.microsoft.bot.builder.TurnContext;
+import com.microsoft.bot.integration.Configuration;
+import org.slf4j.LoggerFactory;
+import java.util.concurrent.CompletableFuture;
+
+public class QnABot extends ActivityHandler {
+    private Configuration configuration;
+
+    public QnABot(Configuration withConfiguration) {
+        configuration = withConfiguration;
+    }
+
+    @Override
+    protected CompletableFuture<Void> onMessageActivity(TurnContext turnContext) {
+        QnAMakerEndpoint qnAMakerEndpoint = new QnAMakerEndpoint() {
+            {
+                setKnowledgeBaseId(configuration.getProperty("QnAKnowledgebaseId"));
+                setEndpointKey(configuration.getProperty("QnAEndpointKey"));
+                setHost(configuration.getProperty("QnAEndpointHostName"));
+            }};
```

Comment:
```suggestion
        QnAMakerEndpoint qnAMakerEndpoint = new QnAMakerEndpoint() {
            {
                setKnowledgeBaseId(configuration.getProperty("QnAKnowledgebaseId"));
                setEndpointKey(configuration.getProperty("QnAEndpointKey"));
                setHost(configuration.getProperty("QnAEndpointHostName"));
            }
        };
```

Reply:
thought the syntax was [like that](https://github.com/southworks/botbuilder-java/pull/61#discussion_r580414135). I'll change it here and in the other portion 👍